src/client/driver/command-executors/execute-action.js

```diff
@@ -124,24 +132,36 @@ function createAutomation (elements, command) {
     // the old eslint version (v1.x.x). We should migrate to v2.x.x
     switch (command.type) {
         case COMMAND_TYPE.click :
+            ensureCommandOffsetOptions(elements[0], command);
```

Comment:
Better call it one time and do nothing if options don't contain `offset` options